source/data_utils.py

```diff
@@ -383,21 +294,20 @@ def plot_magnetogram(amap, file, scale=1, vmin=-2000, vmax=2000, cmap=plt.cm.get
     ppxx = pxx / fszh  # Horizontal size of each panel in relative units
     ppxy = pxy / fszv  # Vertical size of each panel in relative units
     ppadv = padv / fszv  # Vertical padding in relative units
-    ppadv2 = padv2 / fszv  # Vertical padding in relative units
+    # Never used so commented out
```

Comment:
these are not ever used I would delete them but no totally sure what they are for?